webdev3.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <title>Leaflet Map</title>
+    <!-- External Stylesheets -->
+<link rel="stylesheet" href="https://unpkg.com/leaflet@1.5.1/dist/leaflet.css"/>
+    <script src="https://unpkg.com/leaflet@1.5.1/dist/leaflet.js"></script>
+    <h2>Interactive Leaflet Map</h2>
+    <hr>
+    <style>
+        body {
+            padding: 0;
+            margin: 0;
+        }
+        html, body, #map {
+            height: 100%;
+            width: 100%;
+        }</style>
+</head>
+
+<body>
+        <p>GIS 4533: Samuel Sharp</p>
+        <a href="https://leafletjs.com/">I built this with Leaflet</a>
+<div id="map"></div>
+<!-- Add the Leaflet JavaScript library -->
+<script src="https://unpkg.com/leaflet@1.0.3/dist/leaflet.js"></script>    
+
+    <script> 
+    
+    //Create variable to hold map element, give initial settings to map
+    
+    //Add new basemap
+        
+        var map = L.map('map',{ center: [46.77035, -92.12242], zoom: 9});
+    L.tileLayer('https://{s}.tile.thunderforest.com/transport-dark/{z}/{x}/{y}.png', {attribution: '&copy; <a href="http://www.thunderforest.com/">Thunderforest</a>, &copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors'}).addTo(map);
+        
+    </script>
+        
+        
+    <script>
+        //Create point feature for Split Rock Lighthouse
+        var marker = L.marker([47.2000, -91.3670]).addTo(map);
+        
+        marker.bindPopup("This is Split Rock Lighthouse").openPopup();
+        
+    </script>
+    
+    <script>
+        
+         //Create point feature for Enger Tower
+        var marker = L.marker([46.7761, -92.1250]).addTo(map);
+          marker.bindPopup("This is Enger Tower").openPopup();
```

Comment:
Nice organization to your code.
I don't think you need to have every piece in a separate `<script>` tag.